src/controllers/favorites.controllers.js

```diff
@@ -1,16 +1,17 @@
-import favoriteModel from '../models/favorite';
+import Favorite from '../models/favorite';
+import User from '../models/user';
+import House from '../models/house';
 
 export const addToFavorite = async (data) => {
   const { userId, houseId } = data;
 
-  const user = await userService.getById(userId);
-  const house = await houseService.getById(houseId);
+  const user = await User.getById(userId);
```

Comment:
`TypeError: User.getById is not a function`

Reply:
`findById` could probably work but in case an id is incorrect, there will be an unhandled error. You could also use `getById` from the user controllers prepared by Marta but error handling there will also be needed. Patryk prepared error handling in `userUpdated` that could work here as well.